cmd/nodecmd/status.go

```diff
@@ -234,46 +232,95 @@ func printOutput(
-	ux.Logger.PrintToUser(strings.Repeat("=", len(tit)))
+	ux.Logger.PrintToUser(strings.Repeat("=", len(removeColors(tit))))
 	ux.Logger.PrintToUser("")
-	header := []string{"Cloud ID", "Node ID", "IP", "Network", "Avago Version", "Primary Network", "Healthy"}
+	header := []string{"Cloud ID", "Node ID", "IP", "Network", "Role", "Avago Version", "Primary Network", "Healthy"}
 	if subnetName != "" {
 		header = append(header, "Subnet "+subnetName)
 	}
 	table := tablewriter.NewWriter(os.Stdout)
 	table.SetHeader(header)
 	table.SetRowLine(true)
-	for i, ansibleHostID := range ansibleHostIDs {
-		boostrappedStatus := logging.Green.Wrap("BOOTSTRAPPED")
-		if slices.Contains(notBootstrappedHosts, ansibleHostID) {
-			boostrappedStatus = logging.Red.Wrap("NOT_BOOTSTRAPPED")
+	for i, cloudID := range cloudIDs {
+		nodeTypes := []string{}
+		boostrappedStatus := ""
+		healthyStatus := ""
+		nodeIDStr := ""
+		avagoVersion := ""
+		if clusterConf.MonitoringInstance != cloudID {
+			boostrappedStatus = logging.Green.Wrap("BOOTSTRAPPED")
+			if slices.Contains(notBootstrappedHosts, cloudID) {
+				boostrappedStatus = logging.Red.Wrap("NOT_BOOTSTRAPPED")
+			}
+			healthyStatus = logging.Green.Wrap("OK")
+			if slices.Contains(notHealthyHosts, cloudID) {
+				healthyStatus = logging.Red.Wrap("UNHEALTHY")
+			}
+			if clusterConf.IsAPIHost(cloudID) {
+				nodeTypes = append(nodeTypes, "API")
+			} else {
+				nodeTypes = append(nodeTypes, "NODE")
+			}
+			nodeIDStr = nodeIDs[i]
+			avagoVersion = avagoVersions[cloudID]
 		}
-		healthyStatus := logging.Green.Wrap("OK")
-		if slices.Contains(notHealthyHosts, ansibleHostID) {
-			healthyStatus = logging.Red.Wrap("UNHEALTHY")
+		if nodeConfigs[i].IsMonitor {
+			nodeTypes = append(nodeTypes, "MONITOR")
 		}
 		row := []string{
-			hostIDs[i],
-			logging.Green.Wrap(nodeIDs[i]),
-			ansibleHosts[ansibleHostID].IP,
-			clustersConfig.Clusters[clusterName].Network.Name(),
-			avagoVersions[ansibleHostID],
+			cloudID,
+			logging.Green.Wrap(nodeIDStr),
+			nodeConfigs[i].ElasticIP,
+			clusterConf.Network.Name(),
+			strings.Join(nodeTypes, ","),
+			avagoVersion,
 			boostrappedStatus,
 			healthyStatus,
 		}
 		if subnetName != "" {
-			syncedStatus := logging.Red.Wrap("NOT_BOOTSTRAPPED")
-			if slices.Contains(subnetSyncedHosts, ansibleHostID) {
-				syncedStatus = logging.Green.Wrap("SYNCED")
-			}
-			if slices.Contains(subnetValidatingHosts, ansibleHostID) {
-				syncedStatus = logging.Green.Wrap("VALIDATING")
+			syncedStatus := ""
+			if clusterConf.MonitoringInstance != cloudID {
+				syncedStatus = logging.Red.Wrap("NOT_BOOTSTRAPPED")
+				if slices.Contains(subnetSyncedHosts, cloudID) {
+					syncedStatus = logging.Green.Wrap("SYNCED")
+				}
+				if slices.Contains(subnetValidatingHosts, cloudID) {
+					syncedStatus = logging.Green.Wrap("VALIDATING")
+				}
 			}
 			row = append(row, syncedStatus)
 		}
 		table.Append(row)
 	}
 	table.Render()
 }
+
+func removeColors(s string) string {
```

Comment:
you can use https://pkg.go.dev/github.com/pborman/ansi#Strip for this.
no need to hardcode list of colours